src/worker/keyman-api-service.ts

```diff
@@ -0,0 +1,36 @@
+import { keyboardDataObj, KeymanObj } from "./models";
+
+export class KeymanApi {
+  /**
+   * API configs
+   */
+  baseUrl = "https://api.keyman.com/search";
+  /**
+   * Query data to searche only for languages that match
```

Comment:
```suggestion
   * Query data to search only for languages that match
```

That match what? Could you please link to their API documentation?